openpype/hosts/houdini/plugins/create/create_workfile.py

```diff
@@ -30,16 +31,27 @@ def create(self):
         task_name = self.create_context.get_current_task_name()
         host_name = self.host_name
 
+        current_instance_asset = None
+        if current_instance is not None:
+            if AYON_SERVER_ENABLED:
+                current_instance_asset = current_instance.get("folderPath")
+            if not current_instance_asset:
+                current_instance_asset = current_instance.get("asset")
```

Comment:
Wouldn't this technically fail when opening a workfile built with OpenPype in a project converted to AYON now opened in AYON.

The currently existing instance does **not yet** have `folderPath` defined even though `AYON_SERVER_ENABLED` is True?

Also, I wonder whether it makes sense putting this somewhere in a library for now so it can be re-used:
```python
def get_instance_asset(instance):
    if AYON_SERVER_ENABLED and "folderPath" in instance:
        return instance["folderPath"]
    return instance.get("asset")
```

Reply:
I didn't realized I'm working with already created object of `CreatedInstance`.